internal-import-file/import-document/Dockerfile_analysis

```diff
@@ -0,0 +1,17 @@
+FROM python:3.11-alpine
+ENV CONNECTOR_TYPE=INTERNAL_ANALYSIS
```

Comment:
what does this Dockerfile do ? 
I can guess it changes CONNECTOR_TYPE to INTERNAL_ANALYSIS, so the connector type is only INTERNAL_ANALYSIS? it's not INTERNAL_IMPORT_FILE anymore ?

Reply:
to see with platform

Reply:
file to remove @JeremyCloarec